### PR TITLE
Init migration to React 18 and SForms 0.7.0

### PR DESCRIPTION
**Pull request by VojtechLunak**
#26 
`IntelligentTreeSelect` not working properly in s-forms-components storybook.

### CODE REVIEW

File: src/styles/components.css [hunk 1]
```diff
@@ -73,7 +73,7 @@
   min-height: 24px;
 }
 
-.Select-control:hover {
+.css-13cymwt-control:hover {
```
Comment (blcham) [inline src/styles/components.css]: What is this strange class ?

File: src/styles/components.css [hunk 2]
```diff
@@ -87,29 +87,21 @@
   min-width: 2rem;
 }
 
-.type-answer-group .Select {
+.type-answer-group .css-b62m3t-container {
```
Comment (blcham) [inline src/styles/components.css]: What a strange class?

File: src/styles/components.css [hunk 3]
```diff
@@ -87,29 +87,21 @@
...
   margin-top: -6px;
   flex-grow: 1;
   margin-left: 15px;
   min-width: 15rem;
 }
 
-.type-answer-group .Select-control {
+.type-answer-group .css-13cymwt-control {
```
Comment (blcham) [inline src/styles/components.css]: ?

**Review by blcham — APPROVED** [review]
LGTM, just do not understand what are those strange classes. Shouldn't it be somehow more readable/understandable?

File: src/styles/components.css (hunk 1 above)
Comment (VojtechLunak) [reply src/styles/components.css]: fixed in https://github.com/kbss-cvut/s-forms-components/pull/27/commits/4aa64f8cd10479cd7279faf6c14fe22365be6599

**Comment by blcham** [comment]
You did not request review again, but i assume you wanted ... LGTM, can be merged.

**Pull request merged**